src/crawl4ai_mcp.py

```
@@ -4,71 +4,72 @@
 the appropriate crawl method based on URL type (sitemap, txt file, or regular webpage).
 Also includes AI hallucination detection and repository parsing tools using Neo4j knowledge graphs.
+
+Refactored for Phase 3: Streamlined server with separated tool definitions and service layer integration.
 """
-import asyncio
-import concurrent.futures
-import json
+
 import os
-import re
 import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)